core/api/src/graphql/public/types/object/user.ts

```diff
@@ -75,6 +77,22 @@ const GraphQLUser = GT.Object<User, GraphQLPublicContextAuth>({
       },
     },
 
+    mobileSessions: {
+      type: GT.NonNullList(MobileSession),
+      description: "List of mobile sessions",
+      resolve: async (source, args, { user }) => {
+        return Users.listMobileSessions(user.id)
```

Comment:
we must guarantee that this method does not return an error otherwise will block me query. Please add validation or guarantee at app layer that we always return a mobile session